pkg/target/vapp_dto.go

```diff
@@ -13,12 +13,22 @@ func (vapp *VirtualApp) BuildDTO() (*proto.EntityDTO, error) {
 		NewEntityDTOBuilder(proto.EntityDTO_VIRTUAL_APPLICATION, vapp.UUID).
 		DisplayName(vapp.Name)
 
-	if err := vapp.getCommoditiesBought(vAppBuilder); err != nil {
+	err, qpsused, qpscap, rtused, rtcap := vapp.getCommoditiesBought(vAppBuilder)
+	if err != nil {
 		nerr := fmt.Errorf("build VApplication DTO failed: %v", err)
 		glog.Error(nerr.Error())
 		return nil, nerr
 	}
 
+	// For now, just used the average of the underlying Transaction and ResponseTime commodities.
```

Comment:
we can put this block of code into a separate function, which can be called after the total latency/QPS are computed.